### PR TITLE
OP-439 Increase coverage and format Model file

### PR DESCRIPTION
**Pull request by dbmalkovsky**
Coverage before:
image

Coverage after:
image

Also improvements to `equals()` routine to handle more `null` cases.

### CODE REVIEW

File: src/test/java/org/isf/patvac/test/Tests.java [hunk 1]
```diff
@@ -215,6 +221,305 @@ public void testListenerShouldUpdatePatientToMergedWhenPatientMergedEventArrive(
...
+		patientVaccine.setVaccineDate(new GregorianCalendar());
+
+		vaccineTypeIoOperationRepository.saveAndFlush(vaccineType);
+		vaccineIoOperationRepository.saveAndFlush(vaccine);
+		patientIoOperationRepository.saveAndFlush(patient);
+		patVacIoOperationRepository.saveAndFlush(patientVaccine);
+
+		ArrayList<PatientVaccine> patientVaccines = patVacManager.getPatientVaccine(false);
+		assertThat(patientVaccines.get(patientVaccines.size() - 1).getPatName()).isEqualTo(patientVaccine.getPatName());
+	}
```
Comment (mwithi) [inline src/test/java/org/isf/patvac/test/Tests.java:240]: Could be bettere to enforce the "Today" word in the method's signature by persisting at least two objects in different dates?

File: src/test/java/org/isf/patvac/test/Tests.java [hunk 2]
```diff
@@ -215,6 +221,305 @@ public void testListenerShouldUpdatePatientToMergedWhenPatientMergedEventArrive(
...
+		patientVaccine.setVaccineDate(date);
+
+		vaccineTypeIoOperationRepository.saveAndFlush(vaccineType);
+		vaccineIoOperationRepository.saveAndFlush(vaccine);
+		patientIoOperationRepository.saveAndFlush(patient);
+		patVacIoOperationRepository.saveAndFlush(patientVaccine);
+
+		ArrayList<PatientVaccine> patientVaccines = patVacManager.getPatientVaccine(true);
+		assertThat(patientVaccines.get(patientVaccines.size() - 1).getPatName()).isEqualTo(patientVaccine.getPatName());
+	}
```
Comment (mwithi) [inline src/test/java/org/isf/patvac/test/Tests.java:260]: Same here, could it be better to have one object inside and an other one outside the range to be retrieved?

File: src/test/java/org/isf/patvac/test/Tests.java (hunk 1 above)
Comment (dbmalkovsky) [reply src/test/java/org/isf/patvac/test/Tests.java]: Good idea; added second object.

File: src/test/java/org/isf/patvac/test/Tests.java (hunk 2 above)
Comment (dbmalkovsky) [reply src/test/java/org/isf/patvac/test/Tests.java]: Good idea; added second object.

**Comment by mwithi** [comment]
LGTM.
```
Results :

Tests run: 1703, Failures: 0, Errors: 0, Skipped: 3
```

**Pull request merged**